torch/nn/modules/module.py

```diff
@@ -1954,7 +1958,14 @@ def _load_from_state_dict(self, state_dict, prefix, local_metadata, strict,
                     continue
                 try:
                     with torch.no_grad():
-                        param.copy_(input_param)
+                        if assign_to_params_buffers:
+                            # Shape checks are already done above
+                            if isinstance(param, torch.nn.Parameter):
```

Comment:
```suggestion
                            if isinstance(param, torch.nn.Parameter) and not isinstance(input_param, torch.nn.Parameter):
```

I think it would be great to be able to preserve `is` property here!